src/System.IO.FileSystem/src/System/IO/FindPredicates.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    internal static partial class FindPredicates
```

Comment:
Nit: To me personally, calling this FindPredicates made me though that it would have members that were somehow directly related via the type system to FindPredicate, but that's not the case.

Reply:
Open to suggestions here.